clang/lib/Driver/ToolChains/CommonArgs.cpp

```diff
@@ -490,6 +490,17 @@ void tools::AddLinkerInputs(const ToolChain &TC, const InputInfoList &Inputs,
     else
       A.renderAsInput(Args, CmdArgs);
   }
+  if (const Arg *A = Args.getLastArg(options::OPT_fveclib)) {
+    if (A->getNumValues() == 1) {
+      const llvm::Triple &Triple = TC.getTriple();
+      StringRef V = A->getValue();
+      if ((V == "ArmPL") && (Triple.isOSLinux() || Triple.isOSDarwin())) {
```

Comment:
We prefer no parens for `A == "xxx" && `

Reply:
ok